WordPress/src/main/java/org/wordpress/android/ui/accounts/helpers/FetchBlogListWPOrg.java

```diff
@@ -71,154 +78,301 @@ public void fetchBlogList(Callback callback) {
         (new FetchBlogListTask(callback)).executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
     }
 
-    private String getRsdUrl(String baseUrl) throws SSLHandshakeException {
-        String rsdUrl;
-        rsdUrl = ApiHelper.getRSDMetaTagHrefRegEx(baseUrl);
-        if (rsdUrl == null) {
-            rsdUrl = ApiHelper.getRSDMetaTagHref(baseUrl);
-        }
-        return rsdUrl;
-    }
-
     private boolean isHTTPAuthErrorMessage(Exception e) {
         if (e != null && e.getMessage() != null && e.getMessage().contains("401")) {
```

Comment:
Maybe we can just return this since we are not doing anything else in the `if` statement anymore.